README.md

```diff
@@ -0,0 +1,174 @@
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Microverse! <a name="about-project"></a>
+
+**Hello Microverse** is a project to configure linters, HTML & CSS files.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[key_feature_1]**
+- **[key_feature_2]**
+- **[key_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+*Web browser like  <a href='https://www.google.com/chrome/'>Google Chrome</a>
+
+*<a href='https://git-scm.com/'>Git</a>
+
+*<a href='https://github.com/'>GitHub</a>
+
+*<a href='https://code.visualstudio.com/'>Source Code Editor</a>
+
+### Setup
+
+Clone this repository to your desired folder:
+https://github.com/JhonatanErosa/Project-1_Hello-Microverse
+
+
+### Install
+
+Install this dependencies:
+
+npm install
+
+### Usage
+
+To run the project, execute the following command:
+
+*Clone the repo to your pc
+
+*Use the extension <a href='https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer'>Live Server
+</a> or open de .html file 
+
+### Run tests
+
+To run tests, run the following command:
+
+*HTML linters:
+  
+npx hint .
+
+*CSS linters:
+
+npx stylelint "**/*.{css,scss}"
+
+### Deployment
+
+You can deploy this project using:
+
+<a href='https://github.com/'>GitHub</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Jhonatan Ali Caamal Erosa**
+
+- GitHub: [@JhonatanErosa](https://github.com/JhonatanErosa)
+- Twitter: [@erosa_c](https://twitter.com/erosa_c)
+- LinkedIn: [Jhonatan Ali Caamal Erosa](https://www.linkedin.com/in/jhonatan-ali-caamal-erosa-bb3829271/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
```

Comment:
The future features header is also a very important header in the readme. It adds a professional touch to any project you build by showcasing you as a development-oriented developer and keen on improving your projects. I suggest that you add about 3 features you would like to add in the future of your project. Example, you can say ```About Microverse section``` or ```Animations to improve UI/UX``` or ```JavaScript to add some behaviour to the page```